### ISSUE TITLE
ProseMirror transform isn't creating parent nodes anymore.

### ISSUE THREAD

**Issue by torifat** [ProseMirror/prosemirror#785]
### Issue details

Previously when we inserted any node without its parent, ProseMirror used to create it.

For example the following `Schema` (redacted version):
```js
const nodes = {
  nodeA: {
    content: "nodeB+",
    group: "block",
  },
  nodeB: {},
}
```

Prior to `prosemirror-transform@1.0.3` doing a `tr.insert(pos, nodeB.create())` would create `nodeA -> nodeB`. But, now it's a no-op.

Normally we can fix it by wrapping it with `nodeA` but it makes it harder in case of copy-paste. `parseDom` creates `nodeB` and tries to insert it but it doesn't work.

### Steps to reproduce

https://solid-weaver.glitch.me/

### ProseMirror version
prosemirror-transform@^1.0.3

### Affected platforms
Not relevant.

### Screencast
Working in `prosemirror-transform@^1.0.2`
![kapture 2018-04-05 at 13 51 25](https://user-images.githubusercontent.com/208544/38346429-794a7198-38d8-11e8-8227-033d1d79cb34.gif)

**Comment by marijnh** [comment]
That was indeed a regression. Attached patch (released as 1.0.9) fixes it.

**Comment by torifat** [comment]
Thanks 👍
